project_issue_code/__init__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Michael Viriyananda
+#    Copyright 2016 OpenSynergy Indonesia
+#
+#    This program is free software: you can redistribute it and/or modify
```

Comment:
Please use short header like the conventions for all py files: https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py#L1-L3

Thank you in advance!